.env.example

```diff
@@ -0,0 +1,4 @@
+js PORT=4000
+DATABASE_URL="postgresql://boolean_owner:TZ0rnzWdmbu8@ep-morning-cell-a24xw8zh-pooler.eu-central-1.aws.neon.tech/team-dev-sim?sslmode=require"
```

Comment:
This should be empty, it includes a password :)

Reply:
This is an example file, it should only illustrate the needed keys for the project to work with some placeholder values. not real ones.

Reply:
the real information is stored in the `.env` file, which is not committed